CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-- Update iuf to 0.1.11; cray-nls and cray-iuf to 3.1.8
+- Revert csm-latest tag usage for iuf container image (CASMTRIAGE-5738)
+- Update iuf to 0.1.11; cray-nls and cray-iuf to 3.1.8 (CASM-4467)
```

Comment:
There is nothing in here about updating iuf to v0.1.12.

Reply:
Added an entry for CASMINST-6512